device/cuda/src/finding/combinatorial_kalman_filter.cuh

```diff
@@ -46,6 +46,225 @@
 #include <thrust/sort.h>
 #include <thrust/unique.h>
 
+#include <iterator>
+#include <limits>
+
+namespace traccc::cuda {
+
+__global__ inline void gather_best_tips_per_measurement(
```

Comment:
Is this restructed to be used with a CKF or can we make it a more generic ambiguity resolution algorithm at some point?